FrontEnd/main/src/components/landing-page/ellipses/Ellipses.js

```diff
@@ -0,0 +1,40 @@
+import './Ellipses.css';
+
+export const GreenEllipses = () => {
+  return (
+    <div className="ellipses-main">
+      <div className="ellipses-main__green"/>
```

Comment:
Would probably be better to pass type of ellipses as props - and have one generic component Ellipses